src/modules/validation/implementation/merkle-validation.js

```diff
@@ -51,72 +54,22 @@ class MerkleValidation {
         this.logger = logger;
     }
 
-    calculateHash(assertion) {
-        let stringifiedAssertion = assertion;
-        if (typeof assertion !== 'string' && !(assertion instanceof String)) {
-            stringifiedAssertion = sortedStringify(assertion);
-        }
-        const hash = SHA256(stringifiedAssertion);
-
-        return hash.toString();
+    calculateRoot(assertion) {
+        return calculateRoot(assertion);
     }
 
-    calculateRootHash(nquadsArray) {
+    getMerkleProof(nquadsArray, challenge) {
         nquadsArray.sort();
+
         const leaves = nquadsArray.map((element, index) => keccak256(web3.utils.encodePacked(
             keccak256(element),
             index
         )))
         const tree = new MerkleTree(leaves, keccak256, {sortPairs: true})
```

Comment:
Can we also move merkle tree functions into assertion tool to be sure that dkg.js is using the same implementation?

Reply:
Yes, we will move it. It is not used now so I will remove it from here